guides/02_getting-started/04_blog-tutorial.html.md

```diff
@@ -132,7 +132,7 @@ For now, the defaults for the database connection settings (`config/database.rb`
 OK for this tutorial. A sqlite3 database will be used that is stored inside
 `db/sample_blog_development.db`.
 
-To setup environment, set RACK_ENV=enviromentname. This environment can be configured in config/apps.rb as:
```

Comment:
I like the spelling in this commit with the sentence `This environment can be configured in config/apps.rb as:` better than the new one.